examples/ruby/spec/integration/twilio_api_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'spec_helper.rb'
```

Comment:
These are unit tests so they should be under spec/unit, instead of integration, lets keep them separate.